docs/reference/appointment-types.md

```diff
@@ -0,0 +1,1192 @@
+---
+{
+  "pageClass": "reference-page"
+}
+---
+
+# Appointment types
+
+:::::panel
+
+:::: left
+
+You can use the `/v2/vytemes/:nickname/types` endpoints to manage the availabilities of your users.
+
+> This API is an extension of the Vyteme API, that's why all url are prefixed with `/vyteme/:nickname/`. You can refer to the [Vyteme API reference](vytemes.md) to have more informations about Vyteme creation.
+
+::::
+
+:::: right
+
+> ENDPOINTS
+
+<endpoints>
+  <endpoint method="get" path="/v2/vytemes/:nickname/types" href="#list-all-appointment-types"></endpoint>
+  <endpoint method="post" path="/v2/vytemes/:nickname/types" href="#create-appointment-type"></endpoint>
+  <endpoint method="get" path="/v2/vytemes/:nickname/types/:handle" href="#retrieve-appointment-type"></endpoint>
+  <endpoint method="post" path="/v2/vytemes/:nickname/types/:handle" href="#update-appointment-type"></endpoint>
+  <endpoint method="delete" path="/v2/vytemes/:nickname/types/:handle" href="#delete-appointment-type"></endpoint>
+  
+</endpoints>
+::::
+
+## The Appointment type object
+
+::::: panel
+:::: left
+
+<attributes title="Attributes">
+
+<attribute name="vyteme" type="ObjectId">
+
+The `id` of Vyteme page connected with this Appointment type.
+
+</attribute>
+
+<attribute name="availability" type="ObjectId">
+
+The `id` of Availability page connected with this Appointment type.
+
+</attribute>
+
+<attribute name="name" type="string">
+  
+  Name of the Appointment type
+  
+</attribute>
+ 
+<attribute name="handle" type="string">
+  
+Handle is used at the endpoints as a unique identificator of the Appointment type. Created atomatically from the name by urlifying. For example, name `Conference lunch` will be transfomed into `conference-lunch`.
```

Comment:
It's unique for a vyteme page. But several Vyteme pages can have the same 'conference-lunch' appointment type

Reply:
done a004104d92e7775d157df1b7c2050653f7e7f587